eox_nelp/openedx_filters/xapi/filters.py

```diff
@@ -126,32 +130,32 @@ def run_filter(self, transformer, result):  # pylint: disable=arguments-differ
 
         Arguments:
             transformer <XApiTransformer>: Transformer instance.
-            result <Activity>: Object activity for events related to enrollments.
+            result <Activity>: Module or Problem object activity for any kind of event.
 
         Returns:
             Activity: Modified activity.
         """
-        # Following line is hard-coded since this logic has not been tested with other events.
```

Comment:
Why you remove the exit of events different of "edx.grades.problem.submitted"

Reply:
Because now this use the definition type